src/qt/test/test_main.cpp

```diff
@@ -86,32 +86,32 @@ int main(int argc, char* argv[])
     app.setApplicationName("Bitcoin-Qt-test");
     app.createNode(*init);
 
+    int num_test_failures{0};
+
     AppTests app_tests(app);
-    if (QTest::qExec(&app_tests) != 0) {
-        fInvalid = true;
-    }
+    num_test_failures += QTest::qExec(&app_tests);
```

Comment:
[`QTest::qExec`](https://doc.qt.io/qt-5/qtest.html#qExec) does not guarantee return value `1` in case of a test failure, but:
> a value other than 0 if one or more tests failed or in case of unhandled exceptions.

Reply:
Yes, in my testing if there are two failures for instance, it returns 2.